test/Microsoft.AspNet.Hosting.Tests/HostingEngineTests.cs

```diff
@@ -150,6 +154,87 @@ public void MapPath_Windows_Facts(string virtualPath, string expectedSuffix)
             RunMapPath(virtualPath, expectedSuffix);
         }
 
+        [Fact]
+        public void HostingEngine_CreatesDefaultRequestIdentifierFeature_IfNotPresent()
+        {
+            // Arrange
+            HttpContext httpContext = null;
+            var requestDelegate = new RequestDelegate((httpCntxt) =>
```

Comment:
`httpCntxt` --> `innerHttpContext`? The abbreviation looks more like a typo. Also, can remove the parens around it (single-param lambdas don't need parens).